syntax = "proto3";

package books;

service BooksService {
  rpc GetBooks (BooksRequest) returns (BooksResponse) {}
  rpc AddBook (AddBookRequest) returns (AddBookResponse) {}
}

message BooksRequest {
}

message BooksResponse {
  repeated Book books = 1;
}

message AddBookRequest {
  Book book = 1;
}

message AddBookResponse {
  Book book = 1;
}

message Book {
  string id = 1;
  string title = 2;
  string author = 3;
}

